app/dashboard/template.hbs

```diff
@@ -25,15 +25,15 @@
                             <h2 class='col-xs-9'>{{t 'dashboard.title'}}</h2>
                             <div class='m-b-lg col-xs-3'>
                                 <div class='pull-right'>
-                                    <OsfButton
+                                    <BsButton
```

Comment:
That is the plan. but it's deferred since it incurs visual changes. `BsButton` is a drop-inn replacement for `OsfButton` since it subclassed it without changing/overriding anything.